### PR TITLE
Fix/release issues

### PR DESCRIPTION
**Pull request by l-1squared**

### CODE REVIEW

**Comment by l-1squared** [comment]
@manoj-fd Unfortunately, our gradle schedule task decided to run differently now... Thats why there has not been a release recently.

File: jgiven-maven-plugin/build.gradle [hunk 1]
```diff
@@ -21,7 +21,11 @@ task generatePom(type: Copy, dependsOn: copyClasses) {
     }
 }
 
-task generateMavenPlugin(type: CrossPlatformExec, dependsOn: generatePom) {
+Set<Task> findPublishTask(String projectName){
+   rootProject.findProject(projectName).getTasks().findAll {it instanceof PublishToMavenLocal}
+}
+
+task generateMavenPlugin(type: CrossPlatformExec, dependsOn: [generatePom, findPublishTask("jgiven-core"), findPublishTask("jgiven-html5-report")]){
```
Comment (fudler) [inline jgiven-maven-plugin/build.gradle:28]: Can we actually be sure that these two tasks are the only trouble-makers?

Comment (fudler) [reply jgiven-maven-plugin/build.gradle:28]: Well, it's worth a try.

**Review by fudler — DISMISSED** [review]

File: jgiven-maven-plugin/build.gradle (hunk 1 above)
Comment (l-1squared) [reply jgiven-maven-plugin/build.gradle]: Just checked, the setup I wrote indeed put publish tasks on the graph all the time. Rewrote to fix that

**Review by fudler — APPROVED** [review]

**Comment by manoj-fd** [comment]
> @manoj-fd Unfortunately, our gradle decided to schedule tasks differently, making the release build fail. Thats why there has not been a release recently.

@l-1squared Thanks for looking into the issue, any tentative date we can expect the new release, Thanks again !

**Comment by l-1squared** [comment]
@manoj-fd  How about today?

**Pull request merged**